meshroom/core/node.py

```diff
@@ -503,6 +503,9 @@ def __init__(self, nodeType, position=None, parent=None, uid=None, **kwargs):
         self._internalFolder = ""
         self._sourceCodeFolder = ""
 
+        # Dynamic environment variables: defines variables that will be replaced with set values when used in attributes
+        self.dynamicEnvVars = dict()
+
```

Comment:
On second thought, instead of adding a new concept, these 2 variables should probably just be part of the cmdVars and used as ‘{nodeCacheFolder}’ and ‘{nodeSourceCodeFolder}’. So, we don't mix the dynamic variables and the env variables (that are static by definition).

Reply:
Done